controllers/eventController.class.php

```diff
@@ -131,4 +136,26 @@ public function deleteAction($args) {
             Echo "Event not found.";
         }
     }
+
+    public function joinAction($args) {
+        if (isset($args[0])) {
+            $event = Event::findById(intval($args[0]));
+            $event->addUser(intval($_SESSION["user_id"]));
+            header("location:" . WEBROOT . "event/list");
+        } else {
+            http_response_code(404);
+            echo "User not found";
```

Comment:
Je suis désolé de recommenter cette line, mais juste pour le message d'erreur c'est pas plutot event not found ?